SteveChung.md

```diff
@@ -1,23 +1,15 @@
-## This week 7/11 ~ 7/15
-* Firefox front-end:
+## This week 7/18 ~ 7/22
```

Comment:
Can you drop the this week - last week format? We have not been using that for quite some time and rely on commit diff instead. Thanks!